index.html

```diff
@@ -1642,192 +1662,172 @@ <h2>Architectural Aspects of a Web Thing</h2>
             </p>
             <figure id="arch-aspects">
                 <img src="images/wot-abstract-arch.png" />
-                <figcaption>Architectural Aspects of a
-                    Thing</figcaption>
+                <figcaption>Architectural Aspects of a Thing</figcaption>
             </figure>
         </section>
         <section id="sec-interaction-model">
             <h2>Interaction Model</h2>
-            <p>Originally, a Web resource usually represented a
+            <p>
+                Originally, a Web resource usually represented a
                 document on the World Wide Web that can simply be
-                fetched by a client. With the introduction of Web
-                services, resources became more generic with interaction
-                endpoints that can implement any kind of behavior. This
+                fetched by a Web client. With the introduction of Web
+                services, resources became more generic interaction
+                entities that can implement any kind of behavior. This
                 very high level of abstraction makes it hard to provide
-                a loose coupling between applications and the manifold
-                interaction possibilities of resources. As a result,
-                typical API descriptions consist of a static mapping
+                a loose coupling between applications and resources due to 
+                the manifold interaction possibilities. As a result,
+                at the time of writing typical API descriptions consist of a static mapping
                 from an application intent to a resource address,
                 method, request payload structure, response payload
-                structure, and expected errors.</p>
-            <p>The interaction model of WoT introduces an
+                structure, and expected errors.
+                This imposes a tight coupling between Web client and Web service.</p>
+            <p>
+                The <a>Interaction Model</a> of W3C WoT introduces an
                 intermediate abstraction that formalizes the mapping
-                from application intent to concrete protocol bindings,
-                which are used to interact with Web resources. Instead
-                of generic resources, WoT applications use three types
-                of so-called Interaction Affordances: Properties,
-                Actions, and Events. While constraining Web resources
-                into three different behaviors, the three types are
-                still able to cover virtually all functionality found in
-                IoT devices and services.</p>
+                from application intent to concrete protocol operations
+                and also narrows the possibilities how <a>Interaction Affordances</a>
+                can be modeled.
+            </p>
+            <p>
+                In addition to navigation affordances (i.e., Web links),
```

Comment:
The term "navigation affordance" has not been explicitly introduced so far.
We need another sentence that explains what a "navigation affordance" is.

Btw.: It would be good to have an example of two linked things in the TD spec.

Reply:
See text above under Links and second TODO figure